go/test/endtoend/cluster/mysqlctld_process.go

```diff
@@ -139,6 +139,11 @@ func (mysqlctld *MysqlctldProcess) Stop() error {
 
 // CleanupFiles clean the mysql files to make sure we can start the same process again
 func (mysqlctld *MysqlctldProcess) CleanupFiles(tabletUID int) {
+	os.RemoveAll(path.Join(os.Getenv("VTDATAROOT"), fmt.Sprintf("/vt_%010d/data", tabletUID)))
+	os.RemoveAll(path.Join(os.Getenv("VTDATAROOT"), fmt.Sprintf("/vt_%010d/relay-logs", tabletUID)))
+	os.RemoveAll(path.Join(os.Getenv("VTDATAROOT"), fmt.Sprintf("/vt_%010d/tmp", tabletUID)))
+	os.RemoveAll(path.Join(os.Getenv("VTDATAROOT"), fmt.Sprintf("/vt_%010d/bin-logs", tabletUID)))
+	os.RemoveAll(path.Join(os.Getenv("VTDATAROOT"), fmt.Sprintf("/vt_%010d/innodb", tabletUID)))
```

Comment:
I don't think this is needed since we are removing the entire directory on line 147

Reply:
I remember somewhere in past we saw unlinkat error and resulted in partial deletion of VTTablet folder. Therefore I moved to subfolders deletion . But you are right, this might be overkill. Let me move to just top level directory deletion and if we start getting  unlinkat error or "directory is not empty" then we can bring this change back..